pages/FAQ.md

```diff
@@ -0,0 +1,38 @@
+# FAQ
+
+## I have an error/bug with Slimefun
+Ok, first things first, join our [Support Server](https://slimefun.dev/discord) if you aren't in it already and go to #bug-discussions. We need to determine if it is a bug, user error or intended behaviour.
+
+Now, send us the following information (in the #bug-discussions channel):
+1. Run /sf versions and send us a screenshot. We need the exact versions you are using, otherwise, we will not be able to help you.
+"latest" is not helping us at all. So please run that command. (If you don't have access to this command then Shift-Right-Click your Slimefun Guide, you can find your versions in the uppermost middle slot.)
+
+2. Check your console, are there any errors? (If so, then post them via https://pastebin.com/)
+
+3. What do you intend to happen and what is happening?
+
+Once you've sent us all this info then one of our staff or a community member will help you. **Do not ping any role or members**. If some time has passed (15+ mins) then you may ping the helpful role (`@Helpful`). Being impatient and pinging roles/members may get you warned or kicked.
```

Comment:
"Being impatient and randomly tagging members, especially members of staff".

They can't ping any role besides Helpful anyway. And I'd emphasize that especially staff pings will be penalized.